src/network/serveropcodes.cpp

```diff
@@ -155,71 +152,95 @@ const ClientCommandFactory clientCommandFactoryTable[TOCLIENT_NUM_MSG_TYPES] =
+	{ "TOCLIENT_PRIVILEGES",               MTSCMC_AUTH, true }, // 0x41
+	{ "TOCLIENT_INVENTORY_FORMSPEC",       MTSCMC_FORMSPEC, true }, // 0x42
+	{ "TOCLIENT_DETACHED_INVENTORY",       MTSCMC_INVENTORY, true }, // 0x43
+	{ "TOCLIENT_SHOW_FORMSPEC",            MTSCMC_FORMSPEC, true }, // 0x44
+	{ "TOCLIENT_MOVEMENT",                 MTSCMC_PHYSICS, true }, // 0x45
+	{ "TOCLIENT_SPAWN_PARTICLE",           MTSCMC_PARTICLE, true }, // 0x46
+	{ "TOCLIENT_ADD_PARTICLESPAWNER",      MTSCMC_PARTICLE, true }, // 0x47
 	null_command_factory, // 0x48
-	{ "TOCLIENT_HUDADD",                   1, true }, // 0x49
-	{ "TOCLIENT_HUDRM",                    1, true }, // 0x4a
-	{ "TOCLIENT_HUDCHANGE",                1, true }, // 0x4b
-	{ "TOCLIENT_HUD_SET_FLAGS",            1, true }, // 0x4c
-	{ "TOCLIENT_HUD_SET_PARAM",            1, true }, // 0x4d
-	{ "TOCLIENT_BREATH",                   0, true }, // 0x4e
-	{ "TOCLIENT_SET_SKY",                  0, true }, // 0x4f
-	{ "TOCLIENT_OVERRIDE_DAY_NIGHT_RATIO", 0, true }, // 0x50
-	{ "TOCLIENT_LOCAL_PLAYER_ANIMATIONS",  0, true }, // 0x51
-	{ "TOCLIENT_EYE_OFFSET",               0, true }, // 0x52
-	{ "TOCLIENT_DELETE_PARTICLESPAWNER",   0, true }, // 0x53
-	{ "TOCLIENT_CLOUD_PARAMS",             0, true }, // 0x54
-	{ "TOCLIENT_FADE_SOUND",               0, true }, // 0x55
-	{ "TOCLIENT_UPDATE_PLAYER_LIST",       0, true }, // 0x56
-	{ "TOCLIENT_MODCHANNEL_MSG",           0, true }, // 0x57
-	{ "TOCLIENT_MODCHANNEL_SIGNAL",        0, true }, // 0x58
-	{ "TOCLIENT_NODEMETA_CHANGED",         0, true }, // 0x59
-	{ "TOCLIENT_SET_SUN",                  0, true }, // 0x5a
-	{ "TOCLIENT_SET_MOON",                 0, true }, // 0x5b
-	{ "TOCLIENT_SET_STARS",                0, true }, // 0x5c
+	{ "TOCLIENT_HUDADD",                   MTSCMC_HUD, true }, // 0x49
+	{ "TOCLIENT_HUDRM",                    MTSCMC_HUD, true }, // 0x4a
+	{ "TOCLIENT_HUDCHANGE",                MTSCMC_HUD, true }, // 0x4b
+	{ "TOCLIENT_HUD_SET_FLAGS",            MTSCMC_HUD, true }, // 0x4c
+	{ "TOCLIENT_HUD_SET_PARAM",            MTSCMC_HUD, true }, // 0x4d
+	{ "TOCLIENT_BREATH",                   MTSCMC_PLAYERSTAT, true }, // 0x4e
+	{ "TOCLIENT_SET_SKY",                  MTSCMC_ENVIRONMENT, true }, // 0x4f
+	{ "TOCLIENT_OVERRIDE_DAY_NIGHT_RATIO", MTSCMC_ENVIRONMENT, true }, // 0x50
+	{ "TOCLIENT_LOCAL_PLAYER_ANIMATIONS",  MTSCMC_ENTITY, true }, // 0x51
+	{ "TOCLIENT_EYE_OFFSET",               MTSCMC_CAMERA, true }, // 0x52
+	{ "TOCLIENT_DELETE_PARTICLESPAWNER",   MTSCMC_PARTICLE, true }, // 0x53
+	{ "TOCLIENT_CLOUD_PARAMS",             MTSCMC_ENVIRONMENT, true }, // 0x54
+	{ "TOCLIENT_FADE_SOUND",               MTSCMC_AUDIO, true }, // 0x55
+	{ "TOCLIENT_UPDATE_PLAYER_LIST",       MTSCMC_AUTH, true }, // 0x56
+	{ "TOCLIENT_MODCHANNEL_MSG",           MTSCMC_MODCHANNEL, true }, // 0x57
+	{ "TOCLIENT_MODCHANNEL_SIGNAL",        MTSCMC_MODCHANNEL, true }, // 0x58
+	{ "TOCLIENT_NODEMETA_CHANGED",         MTSCMC_MAP, true }, // 0x59
```

Comment:
Same here. I wouldn't put these on the MAP channel. This could be on the same as ADDNODE/REMOVENODE or even a separate one.